tools/.php-cs-fixer/composer.json

```diff
@@ -1,5 +1,5 @@
 {
     "require": {
-        "friendsofphp/php-cs-fixer": "2.*"
+        "friendsofphp/php-cs-fixer": "3.*"
```

Comment:
The `extra.cotor` section in the main `composer.json` file should also be updated (`cotor` is a tool currently developed internally and used at my employer to generate such files). The extra section is authoritative in theory, but since cotor is currently not publicly available, other people have to work directly in the tools files, yes. So this change is correct. So both items must always be touched for now.

Reply:
I set the version to ^3.3 as it is the latest version with full support for php7.2